tests/imageresize.js

```diff
@@ -435,6 +435,90 @@ describe( 'ImageResize', () => {
 		}
 	} );
 
+	describe( 'percent resizing', () => {
+		describe( 'standard image', () => {
+			beforeEach( async () => {
+				await editor.destroy();
```

Comment:
This is odd. Please create the right editor for the specific describe block. Creating 2 editors for no reason is both confusing and slows down the tests.